contracts/Tokens/VTokens/VToken.sol

```diff
@@ -1371,6 +1365,82 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         return (uint(Error.NO_ERROR), actualRepayAmount);
     }
 
+    /**
+     * @dev Allows a borrower to swap his debt between stable and variable mode, or vice versa
+     * @param rateMode The rate mode that the user wants to swap to
+     **/
+    function swapBorrowRateMode(uint256 rateMode) external {
+        /* Fail if swapBorrowRateMode not allowed */
+        comptroller.swapBorrowRateModeAllowed(address(this));
+
+        accrueInterest();
+
+        /* Verify market's block number equals current block number */
+        if (accrualBlockNumber != getBlockNumber()) {
+            // revert SwapBorrowRateModeFreshnessCheck();
+        }
+
+        address account = msg.sender;
+        (, uint256 variableDebt) = borrowBalanceStoredInternal(account);
+        uint256 stableDebt = _updateUserStableBorrowBalance(account);
+        uint256 accountBorrowsNew = variableDebt + stableDebt;
+        uint256 stableBorrowsNew;
+        uint256 averageStableBorrowRateNew;
+
+        if (InterestRateMode(rateMode) == InterestRateMode.STABLE) {
+            require(variableDebt > 0, "vToken: swapBorrowRateMode variable debt is 0");
+
+            stableBorrowsNew = stableBorrows + variableDebt;
+            uint256 stableBorrowRate = stableBorrowRatePerBlock();
+
+            averageStableBorrowRateNew =
```

Comment:
use SafeMath

Reply:
Fix in [Commit](https://github.com/VenusProtocol/venus-protocol/pull/218/commits/47722840b0a50747600bc29da312e66bf352e3a2)